### PR TITLE
Entities card: special row types, weblink

### PR DESCRIPTION
**Pull request by c727**
fix: https://github.com/home-assistant/ui-schema/issues/119

```yaml
          - type: weblink
            url: http://google.com/
            icon: mdi:google
            name: Google
```

### CODE REVIEW

File: src/panels/lovelace/common/process-config-entities.js [hunk 1]
```diff
@@ -7,6 +7,10 @@ export default function processConfigEntities(entities) {
   }
 
   return entities.map((entityConf, index) => {
+    if (typeof entityConf === 'object' && !Array.isArray(entityConf) && entityConf.type) {
```
Comment (balloob) [inline src/panels/lovelace/common/process-config-entities.js]: If we do this let's change the name to reflect that this is a parser for entity row configs

File: src/panels/lovelace/special-rows/hui-weblink-row.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  static get properties() {
+    return {
+      _config: Object
+    };
+  }
+
+  ready() {
+    super.ready();
+    this.addEventListener('click', () => this._handleClick());
```
Comment (balloob) [inline src/panels/lovelace/special-rows/hui-weblink-row.js]: Instead of this, just wrap whole component in "a" tag

**Review by balloob — APPROVED** [review]

**Comment by balloob** [comment]
Ok to merge once it compiles and tests pass

**Comment by c727** [comment]
Ugh, I always copy my changes to my test system xD

**Comment by balloob** [comment]
The entity config parser file was used in a lot of places where we allow entities. You can't just rename to entity row config parser. Nor can you just allow non entity types. I smell a conflation of concerns 🙃

**Pull request merged**